lib/cronofy.rb

```diff
@@ -5,8 +5,8 @@
 module Cronofy
   class Cronofy
 
-    def initialize(client_id, client_secret, token = false)
-      @auth = Auth.new(client_id, client_secret, token)
+    def initialize(client_id, client_secret, token = false, refresh_token = nil)
```

Comment:
Defaulting `token` to `false` seems odd. As it would be a `String` when provided, `nil` is probably a better default.